qiskit_experiments/test/fake_service.py

```diff
@@ -28,6 +28,41 @@
 )
 
 
+# Check if PANDAS package is installed
+try:
+    import pandas as pd
+
+    HAS_PANDAS = True
+except ImportError:
+    pd = None
+    HAS_PANDAS = False
+
+
+def requires_pandas(func: Callable) -> Callable:
+    """Function decorator for functions requiring Pandas.
+
+    Args:
+        func: a function requiring Pandas.
+
+    Returns:
+        The decorated function.
+
+    Raises:
+        QiskitError: If Pandas is not installed.
+    """
+
+    @functools.wraps(func)
+    def decorated_func(*args, **kwargs):
+        if not HAS_PANDAS:
+            raise ImportError(
+                f"The PANDAS package is required to for {func}."
```

Comment:
```suggestion
                f"The pandas python package is required for {func}."
```
They seem to go with all lower case name on their website.